metaci/build/models.py

```diff
@@ -132,8 +142,13 @@ class Build(models.Model):
     release_relationship_type = models.CharField(max_length=50, choices=RELEASE_REL_TYPES, null=True, blank=True)
     release = models.ForeignKey('release.Release', on_delete=models.SET_NULL, null=True, blank=True)
 
+    objects = BuildQuerySet.as_manager()
+
     class Meta:
         ordering = ['-time_queue']
+        permissions = (
+            ('search_builds', 'Search Builds'),
```

Comment:
Yes, I'm going to assign it in the View All Builds group since I don't think we do perm filtering on search at the moment so you really need the global view_builds perm to be able to use search.

Reply:
commit pushed